### PR TITLE
Promotional updates

### PR DESCRIPTION
**Pull request by zbtirrell**
* update readme
* update admin notices
* update description on plugins page

### CODE REVIEW

File: image-widget.php [hunk 1]
```diff
@@ -438,7 +447,7 @@ public function post_upgrade_nag() {
 	 */
 	public function plugin_row_meta( $meta, $file ) {
 		if ( $file == plugin_basename( dirname( __FILE__ ) . '/image-widget.php' ) ) {
-			$meta[] = '<span class="tribe-test">' . sprintf( __( 'Check out our other <a href="%s" target="_blank">plugins</a> including our <a href="%s" target="_blank">Events Calendar Pro</a>!', 'image-widget' ), 'https://theeventscalendar.com/products/?source=image-widget&pos=pluginlist', 'https://theeventscalendar.com/product/wordpress-events-calendar-pro/?source=image-widget&pos=pluginlist' ) . '</span>';
+			$meta[] = '<strong>Coming Soon:</strong> <a href="http://m.tri.be/19ma" target="_blank">' . __( 'Image Widget Plus', 'image-widget' ) . '</a>';
```
Comment (borkweb) [inline image-widget.php]: * `esc_html__()` for Image Widget Plus?
* Add `esc_html__()` for "Coming Soon"

File: image-widget.php [hunk 2]
```diff
@@ -423,8 +423,17 @@ public function post_upgrade_nag() {
 		$version_key = '_image_widget_version';
 		if ( get_site_option( $version_key ) == self::VERSION ) return;
 
-		$msg = sprintf( __( 'Thanks for using the Image Widget! If you like this plugin, please consider <a href="%s" target="_blank">rating it</a> and maybe even check out our premium plugins including our <a href="%s" target="_blank">Events Calendar Pro</a>!', 'image-widget' ), 'http://wordpress.org/plugins/image-widget/?source=image-widget&pos=nag', 'https://theeventscalendar.com/product/wordpress-events-calendar-pro/?source=image-widget&pos=nag' );
-		echo "<div class='update-nag'>$msg</div>";
+		?>
+		<div class="update-nag">
+			<?php esc_html_e( 'Thanks for using the Image Widget by Modern Tribe! If you like these features, you\'ll love what we\'re working on next.', 'image-widget' );?>
+			<br>
+			<?php echo sprintf(
```
Comment (barryhughes) [inline image-widget.php]: Not essential, but `printf()` is a little more concise then `echo sprintf()`

File: image-widget.php [hunk 3]
```diff
@@ -423,8 +423,17 @@ public function post_upgrade_nag() {
...
+				esc_html__( 'Check out the new %1$sImage Widget Plus%2$s!', 'image-widget' ),
+				'<a href="http://m.tri.be/19mb" target="_blank">',
+				'</a>'
+			);?>
```
Comment (barryhughes) [inline image-widget.php]: Space after `;`

File: readme.txt [hunk 4]
```diff
@@ -4,39 +4,67 @@ Donate link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_i
...
-1. [Donate](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4BSPTNFFY6AL6) - if this is generating enough revenue to support our time it makes all the difference in the world.
-1. Support us by buying our Premium plugins. In particular, *check out [Events Calendar Pro](https://theeventscalendar.com/product/wordpress-events-calendar-pro/?src=imagewidget) and [Event Tickets Plus](https://theeventscalendar.com/product/wordpress-event-tickets-plus/?src=imagewidget)*.
+[Check out Image Widget Plus now!](http://m.tri.be/19mg)
 
-[Check us out on GitHub](https://github.com/moderntribe/image-widget) in case you want to add some pull requests or a fork!
+= Pull Requests & Translations =
+
+ to add your pull requests, translations, or a fork. Translations can also be submitted in [our forum here on WordPress.org](https://wordpress.org/support/plugin/image-widget/).
```
Comment (barryhughes) [inline readme.txt]: Start this sentence with a capital letter?

Comment (barryhughes) [inline readme.txt]: Also, seems a little weird (to me) to have _"Translations can also be submitted..."_ **and** below the _"Check us out on GitHub"_ link -- the second link seems preferable to me as it goes straight to the translations project.

**Comment by zbtirrell** [comment]
this is ready for review again

**Comment by borkweb** [comment]
:+1:

**Pull request merged**